cmake/SetupCompilers.cmake

```diff
@@ -40,11 +40,30 @@
 #
 ###############################################################################
 
-if (NOT RAJA_ENABLE_CLANG_CUDA)
-  set(CMAKE_CXX_STANDARD 14)
-else()
-  set(CMAKE_CXX_STANDARD 11)
-endif ()
+set(COMPILERS_KNOWN_TO_PALEOLITHIC_CMAKE AppleClang Clang GNU MSVC)
```

Comment:
Whilst this variable name is hillarious, something more descriptive might be better.

Reply:
Fair point, I'll name it for the version we require.